### DISCUSSION TITLE
How to contribute to this library

### DISCUSSION THREAD

**Discussion by ZLotusRain** [pymupdf/PyMuPDF#4109] [Q&A] [answered]
I found a memory leak  case related to `get_drawings()` and also i found a solution to that, I want to make the contribution and I can't find how to contribute in the documention, so what should i do?

**Comment by JorjMcKie** [comment]
You could resolve a memory leak by changing the documentation?

Anyway:
You can always contribute to the documentation as well by submitting a PR that changes things in `docs` folder.

Or you may want to explain a bit more about what your observation actually is.

**Reply by ZLotusRain** [reply]
sorry，my expression is ambiguous, i mean i can't find the way to contribute, i want to modify the `page.get_drawings()`

**Reply by JorjMcKie** [reply, marked as answer]
This code is available in the `src` folder. You can develop an improvement in a fork of the repository and then submit a PR.

**Reply by ZLotusRain** [reply]
thanks, i've already submitted a PR.
